### DISCUSSION TITLE
Why preprocessing of Sentinel2 clipped images is resulting in blank white images ?

### DISCUSSION THREAD

**Discussion by rvrmohan** [semiautomaticgit/SemiAutomaticClassificationPlugin#245] [Q&A] [answered]
I downloaded Sentinel2 images (except 1,9 & 10 bands) and clipped them to the study area. Then, I preprocessed them. When they are loaded in QGIS, they appear as blank white images. When I see these preprocessed bands in the folder, they are normal and visible properly. What could be the reason for this ?

**Comment by semiautomaticgit** [comment, marked as answer]
Hello, it is an issue related to nodata. You can simply set the stretching of raster in the layer properties setting minimum and maximum values as 0 and 1 respectively

**Comment by rvrmohan** [comment]
Thanks a lot for your prompt reply.

I want to try using the SCP with higher resolution images. I recently
downloaded Resourcesat LISS III images for my study area from BHUVAN (
https://bhuvan-app3.nrsc.gov.in/data/download/index.php) but could not
figureout how to preprocess and convert them. The Metadata file does not
contain such information.

Please suggest.

Thanks,

Ramamohan

**Reply by semiautomaticgit** [reply]
You are welcome @rvrmohan .

Unfortunately there are not specific tools for LISS III images, however you can use the tool Band calc to preprocess images knowing the calibration parameters.
Please read this tutorial as reference https://fromgistors.blogspot.com/2021/08/image-conversion-to-reflectance.html
